driver/helpers.go

```diff
@@ -94,3 +95,53 @@ func createMountPoint(path string, file bool) error {
 func convertBytesToGibiBytes(nBytes int64) int64 {
 	return nBytes / (1024 * 1024 * 1024)
 }
+
+func getNewVolumeSize(capacityRange *csi.CapacityRange) (int64, error) {
```

Comment:
Could you write some tests for this function? (Not integ tests just some "unit" tests to verify the logic)